src/Tizen.NUI/src/public/BaseComponents/ViewAccessibilityEvent.cs

```diff
@@ -83,35 +125,66 @@ public bool Equals(GestureInfoType other)
         /// <summary>
         /// The != operator.
         /// </summary>
+        /// <param name="arg1">The first value</param>
+        /// <param name="arg2">The second value</param>
+        /// <returns>True if GestureInfoTypes are not equal</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public static bool operator !=(GestureInfoType arg1, GestureInfoType arg2)
         {
             return !arg1.Equals(arg2);
         }
 
+        /// <summary>
+        /// Gets the hash code of this baseHandle.
+        /// </summary>
+        /// <returns>The Hash Code</returns>
         [EditorBrowsable(EditorBrowsableState.Never)]
         public override int GetHashCode()
         {
-            return Tuple.Create((int)type, xBeg, xEnd, yBeg, yEnd, (int)state, eventTime).GetHashCode();
+            return Tuple.Create((int)Type, StartPositionX, EndPositionX, StartPositionY, EndPositionY, (int)State, EventTime).GetHashCode();
         }
     }
 
+    /// <summary>
+    /// Accessibility gesture information event arguments.
+    /// </summary>
     [EditorBrowsable(EditorBrowsableState.Never)]
     public class GestureInfoEventArgs : EventArgs
     {
+        /// <summary>
+        /// The gesture information type.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
         public GestureInfoType GestureInfo { get; internal set; }
+
+        /// <summary>
+        /// True if the event is consumed.
```

Comment:
It seems that this information is for <returns> </returns>

Reply:
@kimcinoo 
Hello, did you mean that `public int Consumed { get; set; }` has return value?
I think that this is a property so /<return> tag is not required.
how do you think?

Reply:
OK this is description not only for "get"
This property type is "int", then we can use "True" for description?

Reply:
Actually, this value is the second object of the template below in DALi. :
`using GesturePair = std::pair<Dali::Accessibility::GestureInfo, bool>;`
I'm not sure why the type is 'int' here, not 'bool' value.
I'll ask it to the person in charge and update it in another PR, for both type and description.

Reply:
My interops were designed on SWIG examples were bool was converted to int before passing from c++ to c# layer. That's why this value is represented as int inside nui.